src/test/java/edu/ucla/library/libcal/services/LibCalProxyServiceIT.java

```diff
@@ -0,0 +1,108 @@
+import io.vertx.core.Vertx;
+import io.vertx.core.eventbus.MessageConsumer;
+import io.vertx.junit5.VertxExtension;
+import io.vertx.junit5.VertxTestContext;
+import io.vertx.serviceproxy.ServiceBinder;
+
+/**
+ * Tests for {@link LibCalProxyService}.
+ */
+@ExtendWith(VertxExtension.class)
+@TestInstance(Lifecycle.PER_CLASS)
+public class LibCalProxyServiceIT {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(LibCalProxyServiceIT.class, MessageCodes.BUNDLE);
+
+    /**
+     * The service proxy for getting bearer token for API call
+     */
+    private OAuthTokenService myTokenProxy;
+
+    /**
+     * The service proxy for testing typical client usage.
+     */
+    private LibCalProxyService myServiceProxy;
+
+    /**
+     * Only used for event bus unregistration.
+     */
+    private MessageConsumer<?> myToken;
+
+    /**
+     * Only used for event bus unregistration.
+     */
+    private MessageConsumer<?> myService;
+
+    /**
+     * Sets up the test.
+     *
+     * @param aVertx A Vert.x instance
+     * @param aContext A test context
+     */
+    @BeforeAll
+    public final void setUp(final Vertx aVertx, final VertxTestContext aContext) {
+        ConfigRetriever.create(aVertx).getConfig().compose(config -> {
+            return LibCalProxyService.create(aVertx, config);
+        }).onSuccess(proxy -> {
+            myService = new ServiceBinder(aVertx).setAddress(LibCalProxyService.ADDRESS)
+                    .register(LibCalProxyService.class, proxy);
+            myServiceProxy = LibCalProxyService.createProxy(aVertx);
+
+            aContext.completeNow();
+        }).onFailure(aContext::failNow);
+    }
+
+    /**
+     * Tears down the test.
+     *
+     * @param aVertx A Vert.x instance
+     * @param aContext A test context
+     */
+    @AfterAll
+    public final void tearDown(final Vertx aVertx, final VertxTestContext aContext) {
+        myServiceProxy.close().compose(result -> myService.unregister()).onSuccess(success -> aContext.completeNow())
+                .onFailure(aContext::failNow);
+    }
+
+    /**
+     * Tests that {@link LibCalProxyService#getLibCalOutput(String, String)} returns content from LibCal
+     * API calls.
+     *
+     * @param aVertx A Vert.x instance
+     * @param aContext A test context
+     */
+    @Test
+    public final void testGetLibCalOutput(final Vertx aVertx, final VertxTestContext aContext) {
+        ConfigRetriever.create(aVertx).getConfig().compose(config -> {
+            return OAuthTokenService.create(aVertx, config);
+        }).onSuccess(service -> {
+            final MessageConsumer<?> myTokenService = new ServiceBinder(aVertx).setAddress(OAuthTokenService.ADDRESS)
+                    .register(OAuthTokenService.class, service);
+            myTokenProxy = OAuthTokenService.createProxy(aVertx);
+            myTokenProxy.getBearerToken().compose(token -> {
+                return myServiceProxy.getLibCalOutput(token, "api/1.1/hours/2572");
```

Comment:
Since this is the absolute path part of the URL, I think it should start with a slash.

```suggestion
                return myServiceProxy.getLibCalOutput(token, "/api/1.1/hours/2572");
```

Reply:
Switched slash in code/README/build script